build.gradle

```diff
@@ -159,7 +159,7 @@ def toPackageClasspath = files(
 sourceSets {
     main {
         java {
-            srcDir "src"
+            srcDirs = ["src", "tests/checkers/inference/test", "tests/checkers/inference/model"]
```

Comment:
I think what I really mean is to move those testing framework files into src directory, and follow similar structure as this [directory](https://github.com/opprop/checker-framework/tree/master/framework/src/org/checkerframework/framework/test). Adding whatever tests subdirectories to source sets seems odd for me. And it also needs developers to additionally mark those subdirectories as src directory in their IDE.